pkg/server/bootstrap_server.go

```diff
@@ -87,36 +88,66 @@ func (bsc *bootstrapServer) GetConfig(cr poolRequest) (*runtime.RawExtension, er
 	data, err = ioutil.ReadFile(fileName)
 	if os.IsNotExist(err) {
 		glog.Errorf("could not find file: %s", fileName)
-		return nil, nil
+		return nil, nil, nil
 	}
 	if err != nil {
-		return nil, fmt.Errorf("server: could not read file %s, err: %v", fileName, err)
+		return nil, nil, fmt.Errorf("server: could not read file %s, err: %v", fileName, err)
 	}
 
 	mc := new(mcfgv1.MachineConfig)
 	err = yaml.Unmarshal(data, mc)
 	if err != nil {
-		return nil, fmt.Errorf("server: could not unmarshal file %s, err: %v", fileName, err)
+		return nil, nil, fmt.Errorf("server: could not unmarshal file %s, err: %v", fileName, err)
 	}
 	ignConf, err := ctrlcommon.ParseAndConvertConfig(mc.Spec.Config.Raw)
 	if err != nil {
-		return nil, fmt.Errorf("parsing Ignition config failed with error: %v", err)
+		return nil, nil, fmt.Errorf("parsing Ignition config failed with error: %v", err)
 	}
 
 	appenders := getAppenders(currConf, nil, bsc.kubeconfigFunc)
 	for _, a := range appenders {
 		if err := a(&ignConf, mc); err != nil {
-			return nil, err
+			return nil, nil, err
 		}
 	}
 
+	return mc, &ignConf, nil
+}
+
+// GetConfig fetches the machine conf(type - Ignition) based on the pool request
+func (bsc *bootstrapServer) GetConfig(cr poolRequest) (*runtime.RawExtension, error) {
```

Comment:
GetConfig -> GetIgnConfig

for those of us skimming

Reply:
I agree that this is confusing, however, this comes from https://github.com/arithx/machine-config-operator/commit/7a18cfb59f5bb52587b6543101f26e26496ecc3a. 

I can happily change it -- and I wanted to -- but this PR is doing some heavy lifting already.

Reply:
if possible would love this to be updated.